spec/static_file_handler.cr

```diff
@@ -99,4 +101,33 @@ describe Kemal::StaticFileHandler do
       response.headers["Allow"].should eq("GET, HEAD")
     end
   end
+
+  it "should send part of files when requested (RFC7233)" do
+    %w(POST PUT DELETE HEAD).each do |method|
+      headers = HTTP::Headers{"Range" => "0-100"}
+      response = handle HTTP::Request.new(method, "/dir/test.txt", headers)
+      response.status_code.should_not eq(206)
+      response.headers.has_key?("Content-Range").should eq(false)
+    end
+
+    %w(GET).each do |method|
+      headers = HTTP::Headers{"Range" => "0-100"}
+      response = handle HTTP::Request.new(method, "/dir/test.txt", headers)
+      response.status_code.should eq(206 || 200)
+      if response.status_code == 206
+        response.headers.has_key?("Content-Range").should eq true
+        match = response.headers["Content-Range"].match(/bytes (\d+)-(\d+)\/(\d+)/)
+        match.should_not be nil
+        if match
+          startRange = match[1].to_i {0}
```

Comment:
Why use `CamelCase` here? Lets stick to `snake_case` convention for naming variables.